autofit/messages/abstract.py

```diff
@@ -268,7 +270,7 @@ def _broadcast_natural_parameters(self, x):
             )
 
     def factor(self, x):
-       # self.assert_within_limits(x)
+        # self.assert_within_limits(x)
```

Comment:
Still cant tell if we should be setting up the bounds ofr the factor optimization properly here or not...

Anyway, if the test works this is fine for now.